### ISSUE TITLE
Launchers should be phet-io customizable

### ISSUE THREAD

**Issue by samreid** [phetsims/projectile-data-lab#80]
From the phet-io design, we would like to customize the launch properties of pre-set launchers (within a reasonable range), 
* Average launch speed
* SD of launch speed
* SD of launch angle

Make sure these custom settings wouldn’t force the projectiles to go off screen.

**Comment by matthew-blackman** [comment]
To clarify the PhET-IO design of the launchers, we want to maintain the consistency of 'Mystery launcher' and 'Custom launcher'. An instructional designer should be able to create their own set of Mystery launchers, but they must always be able to be "solved" on the Sources/Measures screen.

A Mystery launcher can be customized as a 'spring', 'pressure' or 'explosion' type, and the speed/speed SD values for those types can be customized within ranges. The angle standard deviation of the Mystery launchers must also be within the same range as the angle stabilizer slider for the Custom launcher.

To summarize, the PhET-IO design will allow for customization of the following:
- Average launch speed for 'spring', 'pressure', 'explosion'
- Standard deviation of launch speed for 'spring', 'pressure', 'explosion'
- Range of allowed angle stabilizer values
- Launch mechanism type ('spring', 'pressure', 'explosion') for each Mystery launcher
- Angle stabilizer values for each Mystery launcher

**Comment by matthew-blackman** [comment]
Everything is implemented in the above commits except for one feature that we are not moving ahead with at this time:
- PhET-IO customization of the range of allowed angle stabilizer values

The allowed range of angle stabilizer values (launch angle SD values) is currently 0-8 degrees. Increasing this beyond 8 would lead to potential view issues with the angular size of the launcher frame. @samreid and I agree that allowing customization within this range still provides a good PhET-IO experience for clients, since they can still customize the angle variability of the mystery launchers.

All of the remaining features described above are implemented as of the last commit. @samreid and I reviewed the code. Closing.

**Comment by phet-dev** [comment]
Reopening because there is a TODO marked for this issue.

**Comment by samreid** [comment]
@matthew-blackman and I removed the TODO since it was not necessary.
